ads/google/doubleclick.js

```
@@ -79,6 +80,69 @@ function doubleClickWithGpt(global, data, gladeExperiment) {
     parseInt(data.overrideHeight || data.height, 10),
   ]];
 
+  // Center the ad in the container.
+  const container = global.document.querySelector('#c');
```

I see, a variety of code relies on this assumption.  OK, then let's please be sure there's a unit test which will fail loudly if id="c" gets changed.